### DISCUSSION TITLE
modifying CSS theme - only returns 404.

### DISCUSSION THREAD

**Discussion by ghost** [ckan/ckan#7268] [Q&A] [answered]
hi, 

I am following the instructions for customising the colour scheme (https://docs.ckan.org/en/2.9/theming/css.html) but when I add example_teme.css in the public folder nothing happens. 

switching to debug stats 404 - suggesting I am dropping the file into the wrong location, but no matter what I try I cannot get a custom colour scheme. 

any thoughts? 

build on Ubuntu 20.04, install via source, other custom work is working fine (ckanext-scheming).

**Comment by ghost** [comment, marked as answer]
This can be closed when convenient.   I found an alternative.

**Reply by IotaLion** [reply]
Hey, sorry if I'm opening this again.
But which alternative did you use?
We standing in front of the same problem.
